assembl/static2/js/app/graphql/mutations/updateLegalNoticeAndTerms.graphql

```diff
@@ -0,0 +1,17 @@
+mutation UpdateLegalNoticeAndTerms(
+  $legalNoticeEntries: [LangStringEntryInput]!
+  $termsAndConditionsEntries: [LangStringEntryInput]!
+) {
+  updateLegalNoticeAndTerms(legalNoticeEntries: $legalNoticeEntries, termsAndConditionsEntries: $termsAndConditionsEntries) {
+    legalNoticeAndTerms {
```

Comment:
legalNoticeAndTerms here is the same content as the LegalNoticeAndTerms.graphql 
You can create a a graphql fragment for this, see AgentProfileInfo.graphql or Post.graphql and how this is used in other queries.

Reply:
In one case the content is 
legalNoticeEntries {
        localeCode
        value
      }
      termsAndConditionsEntries {
        localeCode
        value
      }

and in the other it's 

legalNotice(lang: $lang)
termsAndConditions(lang: $lang)

Reply:
Oh I was looking at the wrong query. LegalNoticeAndTerms is from Cédric, I don't know what it's used for exactly, need to check

Reply:
not TermsAndLegalNotice.graphql, I'm talking about LegalNoticeAndTerms.graphql

Reply:
I will do some work on these queries before merge, it is a mess. `TermsAndLegalNotice` and `LegalNoticeAndTerms`. I could either rename `LegalNoticeAndTerms` to `LegalNoticeAndTermsEntries` or have only one query that gets everything (maybe with fragments)